### DISCUSSION TITLE
Telescope reticle position problem

### DISCUSSION THREAD

**Discussion by NRaiders** [Stellarium/stellarium#2438] [Q&A]
Hi all,

I have downloaded all the necessary software and drivers and have my equipment connect to my laptop and recognised by Stellarium, 

My issue is that - using the Telescope plugin in Stellarium - my telescope appears "in the ground". Any idea why this is the case? Have I put the wrong values into the ASCOM settings?

I'm using a HEQ5 Pro GoTo Mount & Tripod connected to my laptop using ASCOM (EQASCOM) so do not have the hand controller connected.

Note: I'm in the Southern Hemisphere. 

Thank you for any help with this.

![post-322654-0-27934100-1652014602](https://user-images.githubusercontent.com/70859936/167310881-fc186516-0391-4ed1-abe3-2f7a660046e6.jpg)
![post-322654-0-38136500-1652014617](https://user-images.githubusercontent.com/70859936/167310884-38807c79-5a80-46d5-a640-a166fe03cdfa.jpg)
![post-322654-0-71722900-1652014643](https://user-images.githubusercontent.com/70859936/167310885-ac25d75c-9633-42d9-959b-5d4e8ea22863.jpg)
![post-322654-0-88346100-1652014633](https://user-images.githubusercontent.com/70859936/167310886-70918ab6-f886-41d7-8afa-8414267f1069.jpg)

**Comment by gzotti** [comment]
Did you align your scope using its drive/hand control box before connecting from Stellarium?

**Reply by NRaiders** [reply]
I had previously done a Polar Alignment using the hand controller but that was a while ago and I had packed up and unpacked my scope.

I have a ZWO ASI294MC Pro camera connected to my C8 that I have also connected to my laptop.

My plan was to use NINA with its Polar Alignment plugin in order to do Polar Alignment for my mount.

Should I do that first before loading up Stellarium?

**Reply by gzotti** [reply]
The telescope mount must be set up so that it can find objects by itself. Stellarium only sends and reads coordinate values, without knowing your parking position, camera config or anything else. So yes, align before connecting, nightly.

**Reply by NRaiders** [reply]
Ah, thank you.

I will try doing a Polar Alignment with NINA before connecting up to Stellairum.

Unfortunately, due to rain being forecast for the next week where I live, it will be a little while before I have good conditions to do this and test.

**Reply by NRaiders** [reply]
Well, just for a test I decided to try out GSServer, to see how that would go compared to EQMOD.

After setting up GSServer, starting Stellarium, changing the telescope configuration so it looks at GSServer instead of EQMOD, then connecting to the telescope in Stellarium the telescope reticule appeared straight away above the ground.

I hadn't changed anything else (haven't even done a proper polar alignment yet) but use GSServer instead of EQMOD.

So looks like the issue is with EQMOD for some reason. *shrug* Will test again once I get the opportunity to properly polar align my mount to see if it still happens.
